pkg/tsdb/cloudwatch/request_parser_test.go

```diff
@@ -363,26 +389,35 @@ func Test_migrateAliasToDynamicLabel_single_query_preserves_old_alias_and_create
 						"statistic": "Average",
 						"period": "600",
 						"hide": false
-				  }`, tc.inputAlias)))
+				  }`, tc.inputAlias))
+
+			var query QueryJson
+			err := json.Unmarshal(queryJson, &query)
 			require.NoError(t, err)
 
-			migrateAliasToDynamicLabel(queryJson)
+			migrateAliasToDynamicLabel(&query)
+
+			matchedJson := []byte(fmt.Sprintf(`{
+				"Alias":      "%s",
```

Comment:
If it used to marshal to camelCase, we should use the json struct tags to continue marshaling to camelCase.